.github/workflows/twisted_trunk.yml

```diff
@@ -7,31 +7,35 @@ on:
   workflow_dispatch:
   pull_request:
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
```

Comment:
Can you add a comment explaining why we need this please?

Reply:
This is a fairly standard snippet which means that this workflow will only run once for a given PR. See e.g.

https://github.com/matrix-org/sytest/blob/2780655ce1aa81d06e32c34eba758054eeb69a56/.github/workflows/pipeline.yml#L7-L12

Also used in synapse tests.yml.

That said, we don't need this concurrency blob here once we're back to using a cronjob. I'll remove it later.

Reply:
We won't *need* it once merged, but there's no harm in leaving it in -- it will simply prevent the Twisted trunk workflow from running multiple times concurrently for the same branch. Let's pull it out after merging, since it's handy for testing before then.